lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -681,7 +682,9 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
 
     course = get_course_by_id(course_id)
     course_is_cohorted = is_course_cohorted(course.id)
+    teams_enabled = course.teams_enabled
     cohorts_header = ['Cohort Name'] if course_is_cohorted else []
+    teams_header = ['Team Name'] if teams_enabled else []
```

Comment:
No plans to internationalize these column names. Generally data follows
similar strategies to not internationalizing code, for example. You need to
have data with predictable headers to encourage data standards. So, like
most code is written in English, most data headers are in English.

On Thursday, September 17, 2015, Andy Armstrong <notifications@github.com>
wrote:

> In lms/djangoapps/instructor_task/tasks_helper.py
> <https://github.com/edx/edx-platform/pull/9771#discussion_r39752751>:
>
> >      cohorts_header = ['Cohort Name'] if course_is_cohorted else []
> > +    teams_header = ['Team Name'] if teams_enabled else []
>
> I see from the line above that we aren't supporting i18n for the headers,
> but that seems like something we might need to do. I would think that
> instructors at Edraak would want the columns to be in Arabic, for example.
> Is this a known limitation that we will address in the future,
> @explorerleslie <https://github.com/explorerleslie>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9771/files#r39752751>.
>